### PR TITLE
Added armstrong numbers in java

### PR DESCRIPTION
**Pull request by likecs**
Added implementation of Armstrong number in java.

### CODE REVIEW

File: code/mathematical-algorithms/armstrong_numbers/armstrong_numbers.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		Scanner in = new Scanner(System.in);
+
+		System.out.println("Enter a 3-digit number : ");
+		int number = in.nextInt();
+		int a = number % 10;
+		int b = (number / 10) % 10;
+		int c = number / 100;
+
+		if ((a*a*a + b*b*b + c*c*c) == number) {
+			System.out.println(number + " is a armstring number.");
```
Comment (arnavb) [inline code/mathematical-algorithms/armstrong_numbers/armstrong_numbers.java]: You have a typo here. Should be `armstrong`, not `armstring`

File: code/mathematical-algorithms/armstrong_numbers/armstrong_numbers.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		}
+		else {
+			System.out.println(number + " is not a armstring number.");
```
Comment (arnavb) [inline code/mathematical-algorithms/armstrong_numbers/armstrong_numbers.java]: Same typo on this line.

File: code/mathematical-algorithms/armstrong_numbers/armstrong_numbers.java (hunk 1 above)
Comment (likecs) [reply code/mathematical-algorithms/armstrong_numbers/armstrong_numbers.java]: Updated.

**Comment by AdiChat** [comment]
Looks good 👍 

Hope you are enjoying your journey with **OpenGenus**

**Pull request merged**